Shippo/Shipment.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System;
+using System.Collections.Generic;
```

Comment:
If this class doesn't have any change can you please revert it to master version so that there is no diff in this PR?  Same for parcel also.